runtime/compiler/env/VMJ9.cpp

```diff
@@ -9195,13 +9195,44 @@ TR_J9SharedCacheVM::getDesignatedCodeCache(TR::Compilation *comp)
 void *
 TR_J9SharedCacheVM::getJ2IThunk(char *signatureChars, uint32_t signatureLength, TR::Compilation *comp)
    {
-   return (void *)findPersistentThunk(signatureChars, signatureLength);
```

Comment:
There is a future PR where `findPersistentThunk` will be cleaned up. At the moment, two different types of J2I thunks are stored using this, but they are maintained by completely different data structures. The future cleanup PR will address this.